src/components/layout/Cluster/Cluster.tsx

```diff
@@ -92,9 +86,9 @@ const Cluster: FC<ClusterProps> = ({
   >
     <ClusterParent
```

Comment:
Since we are using `gap` now there is no need to have Wrapper and Parent component we should be ok with removing the wrapper and eliminating any connected properties.

Reply:
Removed the wrapper component